app/src/androidTest/java/com/example/habitsmasher/MainActivityTest.java

```diff
@@ -133,6 +139,9 @@ public void navigateToHabitViewFragment() {
         // Enter date
         enterCurrentDateInAddHabitDialogBox();
 
+        // Select days
+        setDaysInAddHabitDialogBox();
```

Comment:
With days of the week being a requirement for a habit now, each test needs to include them now. The default is Monday, Wednesday, and Friday buttons.